aldjemy/orm.py

```diff
@@ -97,6 +97,8 @@ def prepare_models():
             name = model._meta.db_table
             orm.mapper(sa_models[name], table, attrs)
         model.sa = sa_models[name]
+        model.t = tables[name]
+        model.c = model.t.c
```

Comment:
I think this should be left to the developer. At my work we don't allow single letter variable names (safe for `i`), and this would go against it. It's also easy to implement with a mixing, or maybe a metaclass.